assets/js/components/Banner/LearnMoreLink.tsx

```diff
@@ -50,12 +77,6 @@ export default function LearnMoreLink( {
 			{ label }
 		</Link>
 	);
-}
-
-LearnMoreLink.propTypes = {
```

Comment:
Hey @tofumatt, I've just noticed that `LearnMoreLink.propTypes` is actually referenced in a few places outside this file, e.g.:

https://github.com/google/site-kit-wp/blob/5432f5e756c4358ca89e0cd11247817dae034092/assets/js/components/Banner/index.js#L136

Do you think it's worth keeping for this reason, to be removed when we migrate all the files which reference it?

Reply:
I didn't notice that 🤔

I'll leave it for now, fair enough.